sdk/maps/maps-search/review/maps-search.api.md

```diff
@@ -0,0 +1,756 @@
+    LocalStreet = "LocalStreet",
+    // (undocumented)
+    Ramp = "Ramp",
+    // (undocumented)
+    Rotary = "Rotary",
+    // (undocumented)
+    Terminal = "Terminal"
+}
+
+// @public
+export enum KnownSearchAddressResultType {
+    // (undocumented)
+    AddressRange = "Address Range",
+    // (undocumented)
+    CrossStreet = "Cross Street",
+    // (undocumented)
+    Geography = "Geography",
+    // (undocumented)
+    POI = "POI",
+    // (undocumented)
+    PointAddress = "Point Address",
+    // (undocumented)
+    Street = "Street"
+}
+
+// @public
+export enum KnownSearchIndexes {
+    Address = "Addr",
+    CrossStreets = "Xstr",
+    Geographies = "Geo",
+    PointAddresses = "PAD",
+    PointsOfInterest = "POI",
+    Streets = "Str"
+}
+
+// @public
+export type LatLon = [latitude: number, longitude: number];
+
+// @public
+export type LocalizedMapView = string;
+
+// @public
+export class MapsSearchClient {
+    constructor(credential: AzureKeyCredential, options?: MapsSearchClientOptions);
+    constructor(credential: TokenCredential, mapsAccountClientId: string, options?: MapsSearchClientOptions);
+    beginFuzzySearchBatch(requests: FuzzySearchRequest[], options?: FuzzySearchBatchOptions): Promise<BatchPoller<BatchResult<SearchAddressResult>>>;
+    beginReverseSearchAddressBatch(requests: ReverseSearchAddressRequest[], options?: ReverseSearchAddressBatchOptions): Promise<BatchPoller<BatchResult<ReverseSearchAddressResult>>>;
+    beginSearchAddressBatch(requests: SearchAddressRequest[], options?: SearchAddressBatchOptions): Promise<BatchPoller<BatchResult<SearchAddressResult>>>;
+    fuzzySearch(searchQuery: SearchQuery, options?: FuzzySearchOptions & OperationOptions): Promise<SearchAddressResult>;
+    getGeometries(geometryIds: string[], options?: GetGeometriesOptions): Promise<EntityGeometry[]>;
+    getPointOfInterestCategories(options?: GetPointOfInterestCategoriesOptions): Promise<PointOfInterestCategory[]>;
+    resumeFuzzySearchBatch(resumeFrom: string, options?: FuzzySearchBatchOptions): Promise<BatchPoller<BatchResult<SearchAddressResult>>>;
+    resumeReverseSearchAddressBatch(resumeFrom: string, options?: ReverseSearchAddressBatchOptions): Promise<BatchPoller<BatchResult<ReverseSearchAddressResult>>>;
+    resumeSearchAddressBatch(resumeFrom: string, options?: SearchAddressBatchOptions): Promise<BatchPoller<BatchResult<SearchAddressResult>>>;
+    reverseSearchAddress(coordinates: LatLon, options?: ReverseSearchAddressOptions & OperationOptions): Promise<ReverseSearchAddressResult>;
+    reverseSearchCrossStreetAddress(coordinates: LatLon, options?: ReverseSearchCrossStreetAddressOptions): Promise<ReverseSearchCrossStreetAddressResult>;
+    searchAddress(query: string, options?: SearchAddressOptions & OperationOptions): Promise<SearchAddressResult>;
+    searchAlongRoute(query: string, maxDetourTime: number, route: GeoJsonLineString, options?: SearchAlongRouteOptions): Promise<SearchAddressResult>;
```

Comment:
what's the unit of `maxDetourTime`? We should include the unit in the parameter name.

Reply:
It's in seconds. Thanks for  pointing this out. Fixed in d9618a782d2f68cd381b386f662950b5ba95ea9c